builtin/credential/userpass/path_users.go

```diff
@@ -42,23 +43,26 @@ func pathUsers(b *backend) *framework.Path {
 
 			"policies": &framework.FieldSchema{
 				Type:        framework.TypeCommaStringSlice,
-				Description: "Comma-separated list of policies",
+				Description: "(DEPRECATED) Use 'token_policies' instead. If this and 'token_policies' are both specified only 'token_policies' will be used.",
```

Comment:
It might be nice to add a common deprecation text to tokenutil.  It would be more generic but we are going to have to write this everywhere.